test/derivative_tests.jl

```diff
@@ -35,11 +35,8 @@ function test_derivatives(method; args = [], kwargs = [], name::String)
 
         # Interpolation transition points
         for _t in t[2:(end - 1)]
-            if func isa Union{BSplineInterpolation, BSplineApprox,
-                CubicHermiteSpline}
-                fdiff = forward_fdm(5, 1; geom = true)(func, _t)
-                fdiff2 = forward_fdm(5, 1; geom = true)(t -> derivative(func, t), _t)
-            elseif func isa SmoothedConstantInterpolation
+            if func isa Union{SmoothedConstantInterpolation, BSplineInterpolation, BSplineApprox}
+                # TODO fix interpolations
```

Comment:
use `@test_broken` instead?

Reply:
fair.

Reply:
the tricky part is that some of these work (typically if the derivative is the same before vs after). As such, `@test_broken` also fails.